backend/src/main/scala/com/softwaremill/adopttapir/template/ProjectTemplate.scala

```diff
@@ -0,0 +1,88 @@
+package com.softwaremill.adopttapir.template
+
+import com.softwaremill.adopttapir.starter.{StarterConfig, StarterDetails}
+import com.softwaremill.adopttapir.template.sbt.Dependency.{PluginDependency, ScalaDependency}
+import os.RelPath
+
+case class FileTemplate(
+    relativePath: RelPath,
+    content: String
+)
+
+class ProjectTemplate(config: StarterConfig) {
+
+  def getBuildSbt(starterDetails: StarterDetails): FileTemplate = {
```

Comment:
I think to be precise, we don't return a template here, but a generated file (basing on a template)?

Reply:
Yup the naming is miss-leading. 
I will change `FileTemplate` -> `GeneratedFile`. Good catch!